tuple/test/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-add_executable(tuple_test)
+add_executable(tuple_test engagement_test.cpp)
```

Comment:
I don't think we need to add this here since it's in target_sources below

Reply:
Keeping this as is due to Alex's comments.